backends/gstreamer/webrtc.rs

```diff
@@ -168,7 +179,10 @@ impl WebRtcControllerBackend for GStreamerWebRtcController {
         message: &DataChannelMessage,
     ) -> WebRtcResult {
         match self.data_channels.lock().unwrap().get(id) {
-            Some(ref channel) => channel.send(message),
+            Some(ref channel) => match channel {
+                DataChannelEventTarget::Created(ref channel) => channel.send(message),
+                _ => Ok(()),
```

Comment:
It seems a bit worrying that we silently ignore messages here. We could get here by calling create_data_channel then immediately calling send_data_channel_message - should we return an error instead?

Reply:
Nevermind, I was confused.